### PR TITLE
added universal link event launchedFromLink

### PR DESCRIPTION
**Pull request by bijoshtj**

### CODE REVIEW

File: tealeaf/platform/PluginManager.h [hunk 1]
```diff
@@ -40,6 +40,7 @@
 - (void) onPause;
```
Comment (rampr) [inline tealeaf/platform/PluginManager.h:40]: why can't we use handleOpenURL itself ?

Comment (bijoshtj) [reply tealeaf/platform/PluginManager.h:40]: handleOpenURL is invoked when another app is requesting to handle resource.
Let app open from link be separate event.

Comment (rampr) [reply tealeaf/platform/PluginManager.h:40]: That way, safari and google play store are _other_ apps. These are also apps that trigger our app to be opened

Comment (bijoshtj) [reply tealeaf/platform/PluginManager.h:40]: nope. I checked. on opening our app from link, this callback is not getting triggered.

**Review by rampr — APPROVED** [review]

**Review by rohanhc — APPROVED** [review]

**Pull request merged**